### PR TITLE
[FMT] format example/05_glm_second_level

### PR DESCRIPTION
**Pull request by Remi-Gau**
Relates to https://github.com/nilearn/nilearn/issues/2528

Changes proposed in this pull request:

- apply pre-commit hooks to example/05_glm_second_level

### CODE REVIEW

**Review by bthirion — APPROVED** [review]
LGTM

File: examples/05_glm_second_level/plot_second_level_one_sample_test.py [hunk 1]
```diff
@@ -29,34 +31,35 @@
...
 n_subjects = 16
 data = fetch_localizer_contrasts(
-    ['left vs right button press'],
+    ["left vs right button press"],
     n_subjects,
     get_tmaps=True,
     legacy_format=False,
 )
 
+import matplotlib.pyplot as plt
```
Comment (NicolasGensollen) [inline examples/05_glm_second_level/plot_second_level_one_sample_test.py:40]: to be moved below comment

File: examples/05_glm_second_level/plot_second_level_one_sample_test.py [hunk 2]
```diff
@@ -18,6 +18,8 @@
 
 """
 
+
+import itertools
```
Comment (NicolasGensollen) [inline examples/05_glm_second_level/plot_second_level_one_sample_test.py:22]: Was this needed ? I don't see the corresponding removal...

**Review by NicolasGensollen — APPROVED** [review]
Thanks @Remi-Gau !
LGTM, just a couple small suggestions.

File: examples/05_glm_second_level/plot_second_level_one_sample_test.py (hunk 2 above)
Comment (NicolasGensollen) [reply examples/05_glm_second_level/plot_second_level_one_sample_test.py:22]: I meant, I don't see where the initial import statement was before (must be tired... 😅 )

File: examples/05_glm_second_level/plot_second_level_one_sample_test.py [hunk 3]
```diff
@@ -187,37 +191,33 @@
...
-    'Permutation Test\n(Cluster-Size Error Control)',
-    'Permutation Test\n(Cluster-Mass Error Control)',
+    "Parametric Test",
+    "Permutation Test\n(Voxel-Level Error Control)",
+    "Permutation Test\n(Cluster-Size Error Control)",
+    "Permutation Test\n(Cluster-Mass Error Control)",
 ]
 
 fig, axes = plt.subplots(figsize=(8, 8), nrows=2, ncols=2)
-img_counter = 0
```
Comment (Remi-Gau) [inline examples/05_glm_second_level/plot_second_level_one_sample_test.py:202]: itertool was introduced through the refactoring of the nested loop

Comment (NicolasGensollen) [reply examples/05_glm_second_level/plot_second_level_one_sample_test.py:202]: Right ! Thanks !
Thank god it's Friday... 😅

File: examples/05_glm_second_level/plot_second_level_one_sample_test.py (hunk 1 above)
Comment (Remi-Gau) [reply examples/05_glm_second_level/plot_second_level_one_sample_test.py]: done

File: examples/05_glm_second_level/plot_second_level_one_sample_test.py (hunk 2 above)
Comment (Remi-Gau) [reply examples/05_glm_second_level/plot_second_level_one_sample_test.py]: moved it closer to where it is used

**Review by bthirion — APPROVED** [review]
LGTM

**Comment by bthirion** [comment]
Merging, then. Thx !

**Pull request merged**